src/main.tsx

```diff
@@ -1,11 +1,22 @@
-import React from "react";
+import React, { Suspense } from "react";
 import ReactDOM from "react-dom";
+import { QueryClientProvider } from "@tanstack/react-query";
 import "./index.css";
+import "virtual:windi.css";
+
 import App from "./App";
+import { DeskproContextProvider } from "./hooks/deskproContext";
+import { queryClient } from "./query";
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <DeskproContextProvider>
+      <QueryClientProvider client={queryClient}>
+        <Suspense fallback={<h1></h1>}>
```

Comment:
use for fallback `Spinner`

Reply:
![image](https://user-images.githubusercontent.com/54243490/203047285-7cbfb007-e251-446b-8626-99e472b1a48e.png)
kept getting this error with any spinner, left that for now

Reply:
Spinner - it's for example))) some loading component